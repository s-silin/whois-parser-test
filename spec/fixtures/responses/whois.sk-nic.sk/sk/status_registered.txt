Domain:                       google.sk
Registrant:                   Google
Admin Contact:                Google
Tech Contact:                 Google
Registrar:                    FAJN-0002
Created:                      2003-07-24
Updated:                      2018-07-03
Valid Until:                  2019-07-24
Nameserver:                   ns1.google.com
Nameserver:                   ns2.google.com
Nameserver:                   ns3.google.com
Nameserver:                   ns4.google.com
EPP Status:                   ok

Registrar:                    FAJN-0002
Name:                         FAJNOR IP s. r. o.
Organization:                 FAJNOR IP s. r. o.
Organization ID:              46039201
Phone:                        +421.263811927
Email:                        domains@fajnor.sk
Street:                       Krasovsk√©ho 13
City:                         Bratislava
Postal Code:                  851 01
Country Code:                 SK
Created:                      2017-09-01
Updated:                      2018-07-24

Contact:                      Google
Name:                         Domain Administrator
Organization:                 Google Ireland Holdings Unlimited Company
Phone:                        +353.14361000
Email:                        dns-admin@google.com
Street:                       70 Sir John Rogerson's Quay
City:                         Dublin
Postal Code:                  2
Country Code:                 IE
Registrar:                    FAJN-0002
Created:                      2018-01-26
Updated:                      2018-07-16
