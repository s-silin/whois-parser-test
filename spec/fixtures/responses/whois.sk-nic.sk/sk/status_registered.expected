#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2003-07-24}

#updated_on
  %s %CLASS{time}
  %s %TIME{2018-07-03}

#expires_on
  %s %CLASS{time}
  %s %TIME{2019-07-24}


#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.google.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.google.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns3.google.com"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns4.google.com"
